README.md

```diff
@@ -58,10 +85,18 @@ file for details.
 
 [contributors]:                     https://github.com/UKHomeOffice/pttg-ip-hmrc/graphs/contributors
 [pttg-ip-hmrc-access-code]:         https://github.com/UKHomeOffice/pttg-ip-hmrc-access-code
+[pttg-ip-audit]:                    https://github.com/UKHomeOffice/pttg-ip-audit
 [pttg-ip-api]:                      https://github.com/UKHomeOffice/pttg-ip-api
+[eue-api-rps-service]:              https://bitbucket.ipttools.info/projects/EUE-API/repos/eue-api-rps-service/
```

Comment:
I thought we were only going to list dependencies of each service in the readme? Also, although all our GitHub stuff is publicly visible, BitBucket isn't, so I'm not sure if it's right to link to it from here.